openapi-codegen/sbt-plugin/src/main/scala/sttp/tapir/sbt/OpenapiCodegenPlugin.scala

```diff
@@ -30,56 +42,48 @@ object OpenapiCodegenPlugin extends AutoPlugin {
     openapiJsonSerdeLib := "circe",
     openapiValidateNonDiscriminatedOneOfs := true,
     openapiMaxSchemasPerFile := 400,
-    openapiAdditionalPackages := Nil
+    openapiAdditionalPackages := Nil,
+    openapiStreamingImplementation := "fs2",
+    standardParamSetting
   )
 
-  private def codegen = Def.task {
-    val log = sLog.value
-    log.info("Zipping file...")
-    (((
-      openapiSwaggerFile,
-      openapiPackage,
-      openapiObject,
-      openapiUseHeadTagForObjectName,
-      openapiJsonSerdeLib,
-      openapiValidateNonDiscriminatedOneOfs,
-      openapiMaxSchemasPerFile,
-      openapiAdditionalPackages,
-      sourceManaged,
-      streams,
-      scalaVersion
-    ) flatMap {
```

Comment:
Annoyingly it turns out that this syntax was only supported up to 11 elems for some reason (Why 11? 🤷). Took a while to find a refactor of the sbt plugin that would support aggregation of more setting keys, but I think the solution I've settled on here should be fine for arbitrary N settings.

Reply:
11 is 22/2, which is the tuple limit in Scala ;). Honestly, no idea where this comes from :)